client/src/app/fridge/fridge.component.ts

```diff
@@ -0,0 +1,84 @@
+import {Component, OnInit} from '@angular/core';
+import {
+  NgIf,
+  NgFor,
+  UpperCasePipe,
+  CommonModule,
+} from '@angular/common';
+import {FormsModule} from '@angular/forms';
+
+import { Ingredient } from '../ingredient';
+import {INGREDIENTS} from '../mock-ingredients';
+
+@Component({
+  standalone: true,
+  selector: 'app-fridge-component',
+  templateUrl: './fridge.component.html',
+  styleUrls:['./fridge.component.scss'],
+  imports: [
+    CommonModule,
+    FormsModule,
+    NgIf,
+    NgFor,
+    UpperCasePipe,
+  ],
+})
+export class FridgeComponent implements OnInit{
+
+  addedIngredients: Set<string> = this.getLocallyStoredIngredients();
+  //TODO: Replace this list of ingredients with one found in ingredients table once available
+  availableIngredients = [...INGREDIENTS].map(ingredient => ingredient.name).sort();
```

Comment:
Replace with a `FridgeService` call, that way we only have to refactor the service in the future.